packages/cli/src/config/index.ts

```diff
@@ -23,7 +23,7 @@ if (inE2ETests) {
 	globalConfig.publicApi.disabled = true;
 	process.env.SKIP_STATISTICS_EVENTS = 'true';
 	globalConfig.auth.cookie.secure = false;
-	process.env.N8N_SKIP_AUTH_ON_OAUTH_CALLBACK = 'true';
+	process.env.N8N_SKIP_AUTH_ON_OAUTH_CALLBACK = 'false';
```

Comment:
Original intent seems to have been to skip for testing?

Reply:
We have tests cases that rely that it's not skipped, so I think it's this way to make them pass